packages/kotti-ui/source/kotti-user-menu/types.ts

```diff
@@ -1,16 +1,27 @@
-import { KottiNavbar } from '../kotti-navbar/types'
+import { z } from 'zod'
+
 import { SpecifyRequiredProps } from '../types/utilities'
 
 export namespace KottiUserMenu {
-	export type SectionLink = KottiNavbar.SectionLink
-	export type Section = KottiNavbar.Section
+	export const sectionLinkSchema = z.object({
+		title: z.string(),
+	})
+	export type SectionLink = z.infer<typeof sectionLinkSchema>
+
+	export const sectionSchema = z.object({
+		links: z.array(sectionLinkSchema).nonempty(),
+		title: z.string().nullable(),
```

Comment:
does `nullable()` imply `null | undefined`

Reply:
I think the user-app used to pass `undefined` for a title

and `nullable` apparently makes it `| null`

so it's a breaking change, we need this as a note to make changes in the user-app to avoid console errors by a kotti update